app/scripts/modules/core/src/domain/IEntityTags.ts

```diff
@@ -44,8 +46,12 @@ export interface IEntityTags {
 }
 
 export interface IEntityRef {
-  cloudProvider: string;
-  entityType: string;
-  entityId: string;
   [attribute: string]: any;
+  account?: string;
+  accountId?: string;
+  cloudProvider?: string;
+  entityId?: string;
+  entityType?: string;
```

Comment:
I conflated `IEntityRef` with `IOwner` (from `IOwnerOption.owner`).

I'll make a separate interface for `IOwner` and tighten down the `IEntityRef` fields